boards/arm/raspberrypi_pico/raspberrypi_pico_defconfig

```diff
@@ -0,0 +1,7 @@
+CONFIG_SOC_SERIES_RP2XXX=y
+CONFIG_SOC_RP2040=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=125000000
+CONFIG_RP2_FLASH_W25Q080=y
```

Comment:
Should this be a visible symbol ?
Or should it be placed in `raspberrypi_pico/Kconfig.defconfig` instead and be promptless.